### PR TITLE
Fixed a few small bugs, introduced by the Poller tidy up of laf

### PR DESCRIPTION
**Pull request by exarv**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: includes/polling/functions.inc.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ function poll_sensor($device, $class)
     $misc_sensors = array();
     $all_sensors = array();
 
-    foreach (dbFetchRows("SELECT * FROM `sensors` WHERE `sensor_class` = '?' AND `device_id` = ?", array($class, $device['device_id'])) as $sensor) {
+    foreach (dbFetchRows("SELECT * FROM `sensors` WHERE `sensor_class` = ? AND `device_id` = ?", array($class, $device['device_id'])) as $sensor) {
```
Comment (laf) [inline includes/polling/functions.inc.php]: This was actually changed due to some users complaining that queries weren't being quoted so it was doing WHERE `sensor_class` = temperature and failing.

**Comment by laf** [comment]
You also mention in the other PR that this broke unix-agent hddtemps, does this actually fix it as from testing today and even just now it doesn't seem to. I've put another PR in place that does fix hddtemp graphs.

File: includes/polling/functions.inc.php (hunk 1 above)
Comment (exarv) [inline includes/polling/functions.inc.php]: Hmm. strange. I always learned that, when using ? for a string field, you never quote it...

Comment (laf) [inline includes/polling/functions.inc.php]: Ditto and we don't all over the show but for some reason people have been having issues recently and no one's been able to track it down. So unless this is causing you problems then we should leave it for now.

Comment (exarv) [inline includes/polling/functions.inc.php]: I also see now a dbMakeQuery in includes/dbFacile.php, which combines the query and the data. And as far as I can see, it's not using the PDO prepare / execute

**Comment by exarv** [comment]
My patch fixed the graphs and the sensor (health) alert checking for the unix-agent hddtemp.
At least on my installation (last update from about 24 hours ago)
CentOS 7.3.1611, PHP 5.4.16, MariaDB 5.5.52

File: includes/polling/functions.inc.php (hunk 1 above)
Comment (exarv) [inline includes/polling/functions.inc.php]: Well, the code with the quoted '?' does give me problems. Then the hddtemp drop to 0 degrees in my graphs, and I have alerts for alle servers with unix-agent / hddtemp enabled. (temp = 0 C)

Comment (laf) [inline includes/polling/functions.inc.php]: Is that not just because it's broke anyway?

**Comment by laf** [comment]
The issue overall with how this is now is the reliance on running sensors module as well, without this doesn't work so it makes more sense to separate them out as per my pr.

**Pull request merged**